en/debug/system_wide_replay.md

```diff
@@ -1,93 +1,113 @@
-- Add ORB publisher rules file in `build/px4_sitl_default_replay/rootfs/orb_publisher.rules`.
-  This file defines which module is allowed to publish which messages.
+  This will create the build/make output in a separate build directory `build/px4_sitl_default_replay` (so that the parameters don't interfere with normal builds).
+  It's possible to choose any posix SITL build target for replay, since the build system knows through the `replay` environment variable that it's in replay mode.
+- Add ORB publisher rules in the file `build/px4_sitl_default_replay/rootfs/orb_publisher.rules`.
+  This file defines the modules that are allowed to publish particular messages.
   It has the following format:
+  
   ```
   restrict_topics: <topic1>, <topic2>, ..., <topicN>
   module: <module>
   ignore_others: <true/false>
   ```
-  It means that the given list of topics should only be published by `<module>` (which is the command name).
+  This means that the given list of topics should only be published by `<module>` (which is the command name).
   Publications to any of these topics from another module are silently ignored.
-  If `ignore_others` is `true`, then publications to other topics from `<module>` are ignored.
+  If `ignore_others` is `true`, publications to other topics from `<module>` are ignored.
 
   For replay, we only want the `replay` module to be able to publish the previously identified list of topics.
-  So for replaying `ekf2`, the rules file looks like this:
+  So, for replaying `ekf2`, the rules file should look like this:
   
   ```
   restrict_topics: sensor_combined, vehicle_gps_position, vehicle_land_detected
   module: replay
   ignore_others: true
   ```
   
-  This allows that the modules, which usually publish these topics, don't need to be disabled for replay.
+  With this, the modules that usually publish these topics don't need to be disabled for the replay.
 
-- Optional: setup parameter overrides in the file `build/px4_sitl_default_replay/rootfs/replay_params.txt`.
-  This file should contain a list of `<param_name> <value>`, like:
-  
-  ```
-  EKF2_GB_NOISE 0.001
-  ```
-  
-  By default, all parameters from the log file are applied.
-  When a parameter changed during recording, it will be changed as well at the right time during replay.
-  A parameter in the `replay_params.txt` will override the value and changes to it from the log file will not be applied.
-- Optional: copy `dataman` missions file from the SD card to the build directory.
-  Only necessary if a mission should be replayed.
+- _(Optional)_ Setup parameter overrides (see [instructions below](#overriding-parameters-in-the-original-log)).
+- _(Optional)_ Copy a `dataman` mission file from the SD card to the build directory.
+  This is only necessary if a mission should be replayed.
 - Start the replay:
 
   ```sh
   make px4_sitl_default jmavsim
```

Comment:
I'm wondering if we should change to `make px4_sitl gz_x500`. jMAVSim is not getting much love currently. Something to think about for next iteration.